phase/restorer.go

```diff
@@ -35,6 +35,18 @@ func (r *Restorer) Restore(cache Cache) error {
 		return err
 	}
 
+	if r.LayerMetadataRestorer == nil {
+		r.LayerMetadataRestorer = layer.NewDefaultMetadataRestorer(r.LayersDir, false, r.Logger)
```

Comment:
Should "skip layers" not be a configurable option?

Reply:
The idea was to make this change minimally invasive, with no changes to the signature of the public receiver functions.

Reply:
If you want to "skip layers", you can still provide a `LayerMetadataRestorer`.